app/views/users/index.html.erb

```diff
@@ -1,3 +1,56 @@
+<div id="statistics">
+  Statistics!
+  <p>Top Merchants by Quantity: 
+    <% @top_merchants_by_quantity.each do |merchant| %>
```

Comment:
Very minor: is "Statistics!" (with no tags) supposed to be here, or is it an artifact of earlier work?

Reply:
It's just a non-tagged header for the display - can be edited/removed later!